### DISCUSSION TITLE
Off topic - Hetzner account problem

### DISCUSSION THREAD

**Discussion by yoplait** [vitobotta/hetzner-k3s#236] [General]
Hi everyone, and sorry to bother but I am frustrated about our dear Cloud Hetzner

I have been using it for years, with different customers providing solutions like the one we have here and last year, I changed projects and became busy with other public clouds.

Hetzner did some clean-ups, and they cancelled my account, and now is more than a week trying to get someone from Hetzner to help me to retrieve this or be able to operate it as I have a customer whom I am suggesting their cloud for a POC but … unfortunately, nobody in support is answering, and the emails are not getting replies … nor really nice piece when you are suggesting this for production.

Can anyone here directly contact someone on Hetzner to pass the case and get an answer, I would really appreciate it.

Thanks

Juan Carlos Perez
[perezpardojc@gmail.com](mailto:perezpardojc@gmail.com)

**Comment by vitobotta** [comment]
Sorry to hear that. Didn't you receive any notifications/warnings before they closed your account? I don't know anyone in Hetzner but you could try the sub Reddit, because there are often some staff people who reply and *maybe* they could help.

**Comment by yoplait** [comment]
Thanks for that; I am trying in all ways possible I even open a thread on Reddit, 
https://www.reddit.com/r/hetzner/comments/14m6y0o/rejection_of_my_account/

Thanks for your comment @vitobotta
